### PR TITLE
docs: extend observability populating doc

### PR DESCRIPTION
**Pull request by miguelsorianod**
## Description
* Extend observability populating doc
* Minor formatting changes

## Checklist (Definition of Done)

- [ ] All acceptance criteria specified in JIRA have been completed
- [ ] Unit and integration tests added that prove the fix is effective or the feature works (tested against emulated and non-emulated OCM environment)
- [ ] Documentation added for the feature
- [ ] CI and all relevant tests are passing
- [ ] Code Review completed
- [ ] Verified independently by reviewer
- [ ] Required metrics/dashboards/alerts have been added (or PR created).
- [ ] Required Standard Operating Procedure (SOP) is added.
- [ ] JIRA has created for changes required on the client side

### CODE REVIEW

**Comment by machi1990** [comment]
looks good to me, it will be good to get a description on the parameters from @pb82 and @JameelB

File: docs/populating-configuration.md [hunk 1]
```diff
@@ -187,13 +187,49 @@ the previously mentioned Data Plane elements can pull container images from it:
...
 ```
+OBSERVATORIUM_CONFIG_ACCESS_TOKEN="<observatorium-config-access-token> \
+RHSSO_LOGS_CLIENT_ID=<rhsso-logs-client-id> \
+RHSSO_LOGS_CLIENT_SECRET=<rhsso-logs-client-secret> \
+RHSSO_METRICS_CLIENT_ID=<rhsso-metrics-client-id> \
+RHSSO_METRICS_CLIENT_SECRET=<rhsso-metrics-client-secret> \
 make observatorium/setup
 ```
 
+The description of the previously shown parameters are:
```
[the remark this answers is not in the document]
Comment (machi1990) [reply docs/populating-configuration.md:224]: /cc @JameelB ^

[the remark this answers is not in the document]
Comment (JameelB) [reply docs/populating-configuration.md:224]: May be good to note that the `observatorium_config_access_token` should have read access to the config repo.

The `rhsso_logs_*` should have logs write permission and the `rhsso_metrics_*` should have metrics write permission. They're both used for pushing logs/metrics to Observatorium. @pb82 may be able to add more details to that though.

**Comment by miguelsorianod** [comment]
Hi @pb82,

When you have some time could you take a look expand the documentation around this so I can include it?

Thanks

**Comment by machi1990** [comment]
Ping @pb82 @miguelsorianod to get the status of the PR.

**Comment by pb82** [comment]
@machi1990 @miguelsorianod apologies, will get it done today

**Comment by machi1990** [comment]
> @machi1990 @miguelsorianod apologies, will get it done today

Thanks @pb82

**Comment by machi1990** [comment]
Fixes https://github.com/bf2fc6cc711aee1a0c2a/ffm-project/issues/2

**Comment by pb82** [comment]
@machi1990 @miguelsorianod I've added the details for the Observability Parameters.

File: docs/populating-configuration.md [hunk 2]
```diff
@@ -187,13 +187,48 @@ the previously mentioned Data Plane elements can pull container images from it:
...
+* OBSERVATORIUM_CONFIG_ACCESS_TOKEN: A GitHub token required to fetch the configuration from a private git repository. More details can be found [here](https://github.com/redhat-developer/observability-operator#adding-requisite-resources).
```
Comment (miguelsorianod) [inline docs/populating-configuration.md:225]: Does it need to be a `private` git repository?

Comment (machi1990) [reply docs/populating-configuration.md:225]: I do not think it need to be a private repository. 
Would you agree @pb82 ?

**Comment by machi1990** [comment]
It will be good to get this content in once the question of @miguelsorianod on whether the observability resource repo needs to be a private repository has been answered.

**Comment by machi1990** [comment]
bumping this PR...

**Review by machi1990 — APPROVED** [review]

**Pull request merged**